tests/integration/test_crunch_cube.py

```diff
@@ -1190,9 +1189,9 @@ def test_fruit_x_pets_proportions_by_col(self):
     def test_pets_x_fruit_proportions_by_cell(self):
         cube = CrunchCube(FIXT_PETS_X_FRUIT)
         expected = np.array([
-            [.4285714, .5384615],
-            [.48, .4074074],
-            [.5217391, .5531915],
+            [0.15, 0.35],
+            [0.15189873, 0.27848101],
+            [0.17142857, 0.37142857],
```

Comment:
👍matches R output for this cube. As a warning: this does not use [the cube added](https://github.com/Crunch-io/crunch-cube/blob/66d21b80291a4188d43095cc45980bf4d5966b23/tests/integration/fixtures/cubes/pets-x-fruit-hs.json) in this pull request.